```
Building ms-gdk:x64-windows-static@2410.1.1897...
CMake Error at scripts/cmake/vcpkg_check_linkage.cmake:24 (message):
  This port can only build as a dynamic library, but the triplet selects a
  static library and a static CRT.  Building a dynamic library with a static
  CRT creates conditions many developers find surprising, and for which most
  ports are unprepared.  Therefore, vcpkg fails rather than changing
  VCPKG_LIBRARY_LINKAGE to dynamic.

  Consider choosing a triplet that sets VCPKG_CRT_LINKAGE to dynamic.  For
  more information, explicitly requesting this configuration in a custom
  triplet, please see
  https://learn.microsoft.com/vcpkg/maintainers/functions/vcpkg_check_linkage?WT.mc_id=vcpkg_inproduct_cli#notes

  If you can edit the port calling vcpkg_check_linkage that emits this
  message, consider adding !(static & staticcrt) to the "supports" expression
  so that this combination can fail early.
```

--- a/ports/ms-gdk/portfile.cmake
+++ b/ports/ms-gdk/portfile.cmake
@@ -1,6 +1,6 @@
 set(GDK_EDITION_NUMBER 241001)
 
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
+set(VCPKG_LIBRARY_LINKAGE dynamic)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "https://www.nuget.org/api/v2/package/Microsoft.GDK.PC/${VERSION}"

--- a/versions/m-/ms-gdk.json
+++ b/versions/m-/ms-gdk.json
@@ -4,11 +4,6 @@
       "git-tree": "e8fba173ede65a17cd157e774755e8cf949b72d1",
       "version": "2410.1.1897",
       "port-version": 0
-    },
-    {
-      "git-tree": "c875a3aa6e19fbaa6582d73179af2c6454ec8bf3",
-      "version": "2410.0.1874",
-      "port-version": 0
     }
   ]
 }

--- a/versions/m-/ms-gdk.json
+++ b/versions/m-/ms-gdk.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "e8fba173ede65a17cd157e774755e8cf949b72d1",
+      "git-tree": "8a9d6953b554c94d7a8fbbf7f1b38b432c500787",
       "version": "2410.1.1897",
       "port-version": 0
     }

